.scripts/add_missing_translation_keys.sh

```
@@ -0,0 +1,215 @@
+# Check if Node.js is available
+if ! command -v node &> /dev/null; then
+    echo -e "${RED}Error: Node.js is not installed or not in PATH${NC}"
+    exit 1
+fi
+
+# Check if jq is available (optional but recommended)
+if ! command -v jq &> /dev/null; then
+    echo -e "${YELLOW}Warning: jq is not installed. Install it for better performance: brew install jq${NC}"
+    USE_JQ=false
+else
+    USE_JQ=true
```

Shellcheck (0.11.0)

[warning] USE_JQ appears unused. Verify use (or export if used externally).

(SC2034)